### PR TITLE
feature/PROJ-172: fix according to comments to PR 64

### PR DESCRIPTION
**Pull request by AramZakaryan**
# Changes according to the comments to pull request 64 

# Overview of pull request 64 

The pull request 64 introduces a set of components for building a customizable dropdown menu using @radix-ui/react-dropdown-menu primitives.

## The PR includes the following components:

**DropdownMenu:** Root component for the dropdown.
**DropdownMenuTrigger:** Component that triggers the dropdown menu.
**DropdownMenuContent:** Wrapper for the dropdown items, supporting an optional label prop.
**DropdownMenuItem:** Component for individual dropdown menu items.

# Usage
Example usage
``` 
<DropdownMenu>
  <DropdownMenuTrigger> Trigger </DropdownMenuTrigger>
  <DropdownMenuContent label={'Label'}>
    <DropdownMenuItem> Item1 </DropdownMenuItem>
    <DropdownMenuItem> Item2 </DropdownMenuItem>
    <DropdownMenuItem> Item3 </DropdownMenuItem>
  </DropdownMenuContent>
</DropdownMenu>
```
image

### CODE REVIEW

File: src/components/ui/scroll/scroll.module.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
+@mixin background-transition {
+  transition: background-color 300ms ease-out;
+}
+
```
Comment (NoName-ForTeam) [inline src/components/ui/scroll/scroll.module.scss]: вынеси в _mixins.scss  и заюзай уже локально

File: src/components/ui/dropdown/dropdown.tsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import * as DropdownMenuPrimitive from '@radix-ui/react-dropdown-menu'
 import styles from './dropdown.module.scss'
-import { clsx } from 'clsx'
-import { ComponentPropsWithoutRef, ElementRef, forwardRef } from 'react'
+import {clsx} from 'clsx'
+import {ComponentPropsWithoutRef, ElementRef, forwardRef} from 'react'
 import s from '@/components/ui/dropdown/dropdown.module.scss'
```
Comment (NoName-ForTeam) [inline src/components/ui/dropdown/dropdown.tsx]: ./dropdown.module.scss'

**Comment by NoName-ForTeam** [comment]
ты в папку dropdown add index.ts ?

File: src/components/ui/scroll/scroll.module.scss (hunk 1 above)
Comment (AramZakaryan) [reply src/components/ui/scroll/scroll.module.scss]: сделал

File: src/components/ui/dropdown/dropdown.tsx (hunk 2 above)
Comment (AramZakaryan) [reply src/components/ui/dropdown/dropdown.tsx]: поменял

**Review by Lelik7777 — APPROVED** [review]

**Pull request merged**